docker/checkout.bash

```diff
@@ -36,9 +36,9 @@ while [[ $# -gt 0 ]]; do
 done
 
 if [[ "$BRANCH" = "develop" ]]; then
-      git clone https://github.com/usdot-fhwa-stol/carma-msgs.git ~/src/carma-msgs --branch carma-system-3.7.0 --depth 1
-      git clone https://github.com/usdot-fhwa-stol/carma-utils.git ~/src/carma-utils --branch carma-system-3.7.0 --depth 1
+      git clone https://github.com/usdot-fhwa-stol/carma-msgs.git ~/src/carma-msgs --branch carma-system-3.8.0 --depth 1
```

Comment:
Should these be develop branch? I think this is needed for the develop builds right? Will it cause a merge conflict when we merge back to develop after release?